client/src/app/components/audit-template/create-audit-template/create-audit-template.component.html

```diff
@@ -126,9 +126,13 @@
     </mat-card-content>
   <mat-card-footer>
     <div class="center-aligned">
-      <div class="button-side-by-side">
+      <div class="button-side-by-side" *ngIf="disabled">
+        <button mat-stroked-button id="edit-template-button" class="save-button-bg" (click)="beginEdit()">
+          EDIT</button><!-- if checks throws error then do not send data -->
+      </div>
```

Comment:
Can we remove the Edit button since the component manages audit creation?
I really don't like that it calls an empty function

Reply:
In order to avoid HTML duplication, I need to keep this button here so that I could use it with the edit template component.